assets/js/script.js

```diff
@@ -94,11 +91,28 @@ function uploadFiles(){
 }
 
 
-$(document).ready(function(){
-    var files;
+function restoreForm(event) {
+    event.preventDefault();
     
+    chat_div.empty();
+    users_div.empty();
+    logo_nav.removeClass('nav-back');
+    back_nav.hide();
+    upload_prompt_div.show();
+}
+
+
+$(document).ready(function(){    
     $('form').on('submit', uploadFiles);
     $('input[type=file]').on('change', prepareUpload);
-
+    $('nav').on('click', '.nav-back', restoreForm);
```

Comment:
Use ```.click``` trigger instead of ```.on``` trigger
Reference: https://api.jquery.com/click/

Reply:
I've used `.on` since `.click` does not allow specifying descendants. Since `.nav-back` is attached to `.navbar-brand` after page load, this method insures it will trigger the event as well. I'll adjust this depending on how `.navbar-brand` should operate, as per comments above.

Reply:
Why do you need to specify descendant inside click, you can do that in the selection ($)

Reply:
At page load the only DOM element that matches the `$('.nav-back')` selector would be the back button. This method allows for elements added or changed later on to trigger the event.

I'll change it to `.click` now, since all intended elements will now be present from the start.